src/app/gui/glLUTwidget.cpp

```diff
@@ -464,7 +464,7 @@ GLLUTWidget::GLLUTWidget(LUTChannelMode mode, QWidget *parent) : QGLWidget(paren
   addAction(actionUndo);
 
   actionRedo = new QAction(this);
-  actionUndo->setEnabled(false);
+  actionRedo->setEnabled(false);
```

Comment:
Is this a bug fix?

Reply:
yes, looks like a copy paste error. actionUndo was disabled before already.